src/assertions/elementRegresses.js

```diff
@@ -7,7 +7,10 @@ exports.assertion = class ElementRegresses {
         this.message = `Deviance regression (pass): <${selector}> comparison passed`;
         this.expected = threshold || this.api.globals.deviance.regression.threshold;
         this.value = (result) => {
-            result.toString = () => result.message;
+            if (hasProperty(result, 'message')) {
```

Comment:
@awilson82-vp can you let me know when you have updated this to handle the object issue we spoke about earlier.